metrics.py

```diff
@@ -182,25 +178,12 @@ def get_template_data(self):
     # Memcache doesn't support saving values > 1MB. Break up features into chunks
     # and save those to memcache.
     if self.MODEL_CLASS == models.FeatureObserver:
-      keys = models.get_chunk_memcache_keys(
-          self.PROPERTY_CLASS.all(), self.MEMCACHE_KEY)
-      logging.info('looking for keys %r' % keys)
-      properties = ramcache.get_multi(keys)
-      logging.info('found chunk keys %r' % (properties and properties.keys()))
-
-      # TODO(jrobbins): We are at risk of displaying a partial result if
-      # memcache loses some but not all chunks.  We can't estimate the number of
-      # expected cached items efficiently.  To counter that, we refresh
-      # every 30 minutes via a cron.
+      properties = ramcache.get(self.MEMCACHE_KEY)
```

Comment:
This is pretty minor, but for consistency sake, we might want to refactor this CONST to be RAMCACHE_KEY to avoid any confusion about the backing service.